include/net/coap.h

```diff
@@ -26,6 +26,8 @@
 #include <net/net_ip.h>
```

Comment:
Please follow the commit message style for networking components. So the subject should read `net: coap: ....`

Reply:
done